src/client/src/rt-util/getEnvironment.ts

```diff
@@ -0,0 +1,8 @@
+export function getEnvironment(): string | undefined {
+    const serviceUrl = process.env.REACT_APP_BROKER_HOST || window.location.host;
+
+    if (serviceUrl.includes('localhost')) return 'localhost';
+
+    const envMatch = /web-(?<env>[a-zA-Z]+)\.adaptivecluster\.com/.exec(serviceUrl)
+    return envMatch ? envMatch['groups'].env : undefined
+}
```

Comment:
Note: Need to configure `prettier` and/or `tslint` to always add a new line at EOF to prevent weird git diffs